chromium_src/chrome/browser/ui/webui/chrome_web_ui_controller_factory.cc

```diff
@@ -47,7 +47,7 @@ class BraveDataSource : public content::URLDataSource,
       const std::string& path,
       const content::ResourceRequestInfo::WebContentsGetter& wc_getter,
       const GotDataCallback& callback) override {
-    GURL url = GURL("file:///" + path);
+    GURL url = GURL("brave:///" + path);
```

Comment:
should this be `brave:///` or just `brave://`? https://github.com/brave/muon/pull/628/files#diff-9fe2133d0da6f7a6b87c822e193e342aR52 is different than here

Reply:
I think it still needs to be `brave:///` because otherwise it would be a relative url. `brave` is basically just an alias for `file` that can only be used in certain contexts

Reply:
@darkdh I guess you could look at the url it generates to see if it has 3 slashes or 4

Reply:
3 slashes.
it is like `brave:///Volumes/browser-laptop-bootstrap/src/browser-laptop/app/extensions/brave/index-dev.html` with path `Volumes/browser-laptop-bootstrap/src/browser-laptop/app/extensions/brave/index-dev.html`

Reply:
sgtm